src/delay.js

```diff
@@ -124,7 +122,7 @@ p5.Delay.prototype = Object.create(Effect.prototype);
  *                               below the lowPass will be part of the
  *                               delay.
  */
-p5.Delay.prototype.process = function (src, _delayTime, _feedback, _filter) {
+process (src, _delayTime, _feedback, _filter) {
```

Comment:
I think every line from here until the end of the `class` needs indentation to show that it's part of the `class`. Surprised our linter didn't catch this!